contracts/sdk/utils/tsconfig.base.json

```diff
@@ -0,0 +1,20 @@
+{
+  "include": ["./index.ts"],
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": true,
+    "declarationMap": true,
+    "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
```

Comment:
does this include `downLevelIteration: true` ?

Reply:
I don't think so, but it looks like adding `downLevelIteration: true` is a good call so going to add that to all package.jsons. 

Also it might be worth putting all of the SDKs into a common subdir like `/public-sdks` so that they can all share a base tsconfig (BUT they would all still need their own base configs to include the correct files). This is a re-org that we can hold off on for now until we have more evidence that each SDK won't need more individual tsconfigs to the point where it wouldn't make sense